src/server/contentSecurityPolicy.js

```diff
@@ -104,6 +104,8 @@ export default {
       'nrk.no',
       '*.vg.no',
       'vg.no',
+      'http://vg.no',
```

Comment:
Vg er vel bare https?

Reply:
Om vi skal kunne løse denne embedden omtalt i kortet https://trello.com/c/gKeyYJ5C/584-vg-godkjent-embed-som-ikke-fungerer så må vi støtte `http` også.

Reply:
Men det vil jo ikke fungere i de forskjellige miljøene? Alle iframe src må være HTTPS, ikke HTTP siden vår side er HTTPS. For det andre, redirecter http://vg.no deg til https://vg.no.

Reply:
```Mixed Content: The page at 'https://ndla-frontend.staging.api.ndla.no/subjects/subject:1/topic:1:172733/topic:1:172601/resource:1:157741' was loaded over HTTPS, but requested an insecure resource 'http://www.vg.no/spesial/2014/stillesitting/'. This request has been blocked; the content must be served over HTTPS.```

Reply:
Så jeg tror ikke feilen ligger i CSP rett og slett.